pkgs/by-name/ms/msbuild-structured-log-viewer/package.nix

```diff
@@ -14,13 +14,13 @@
 }:
 buildDotnetModule (finalAttrs: rec {
```

Comment:
Wut? This is really weird, `finalAttrs` is meant to avoid `rec`

Reply:
`finalAttrs.version` can cause issues when overriding (see #310373). I don't think there is a final consensus on what to use when, but the rule I've made for myself is to only use `finalAttrs` when it would actually benefit overriding.

Reply:
I'm not sure I agree, but I also don't care enough in this case, so fine by me.